docs/api/transformations.rst

```diff
@@ -32,6 +32,8 @@ Transformations
     identity
     keep_params_nonnegative
     NonNegativeParamsState
+    normalize_by_update_norm
+    NormalizeByUpdateNormState
```

Comment:
please also add them below, around line 176

Reply:
Added in [b849fc6](https://github.com/google-deepmind/optax/pull/958/commits/b849fc63fd13fcf5a5d47a175e2e5041ebf9b7b7)